var/spack/repos/builtin/packages/py-cleo/package.py

```diff
@@ -12,8 +12,13 @@ class PyCleo(PythonPackage):
     homepage = "https://github.com/sdispater/cleo"
     pypi = "cleo/cleo-0.8.1.tar.gz"
 
+    version("1.0.0a5", sha256="097c9d0e0332fd53cc89fc11eb0a6ba0309e6a3933c08f7b38558555486925d3")
     version("0.8.1", sha256="3d0e22d30117851b45970b6c14aca4ab0b18b1b53c8af57bed13208147e4069f")
```

Comment:
```suggestion
    version("0.8.1", sha256="3d0e22d30117851b45970b6c14aca4ab0b18b1b53c8af57bed13208147e4069f", preferred=True)
```
We don't want to install an alpha release by default